src/main/java/com/google/opensesame/servlets/PersonObject.java

```diff
@@ -0,0 +1,42 @@
+package com.google.opensesame.servlets;
+
+import java.util.ArrayList;
+
+public class PersonObject {
+  private String name;
+  private String description;
+  private ArrayList<String> interestTags;
+  private String gitHubID;
+
+  public PersonObject(String nameInput, String gitHubIDInput) {
+    this(nameInput, gitHubIDInput, "", new ArrayList<String>());
+  }
+
+  public PersonObject(String nameInput, String gitHubIDInput, String descriptionInput) {
+    this(nameInput, gitHubIDInput, descriptionInput, new ArrayList<String>());
+  }
+
+  public PersonObject(
+      String nameInput, String gitHubIDInput, String descriptionInput, ArrayList<String> tagInput) {
+    name = nameInput;
+    gitHubID = gitHubIDInput;
+    description = descriptionInput;
+    interestTags = tagInput;
+  }
```

Comment:
Same comment as above

Reply:
Also look at my comment above about using the builder pattern to reduce the number of constructors.